lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -19,22 +19,36 @@
                 '<span class="member-count"><%= membership_message %></span>' +
                 '<ul class="list-member-thumbs"></ul>'
             ),
+            profileImageTemplate: _.template(
+                '<li class="item-member-thumb"><img alt="<%- username %>" src="<%- profile_image_url %>"></img></li>'
+            ),
 
             initialize: function (options) {
                 this.maxTeamSize = options.maxTeamSize;
             },
 
             render: function () {
-                var memberships = this.model.get('membership'),
+                var totalMemberships = _(this.model.get('membership'))
```

Comment:
An alternative would be just to go back to ``memberships`` and remove ``displayableMemberships`` as it is only used once.